src/test/java/elizCurtnazar/tests/tema8/teste/BaseTest.java

```diff
@@ -0,0 +1,40 @@
+package elizCurtnazar.tests.tema8.teste;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import teofilursan.driver.BrowserManager;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BaseTest {
+
+    protected ChromeDriver driver;
+
+    @BeforeMethod(groups = "desktop")
+    public void initChromeBrowser() {
+        driver = BrowserManager.createChromeDriver();
+//        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        driver.manage().window().maximize();
+        System.out.println("Executed before method");
+
+    }
+    @BeforeMethod(groups = "mobile")
+    public void initChromeBrowserForMobile() {
+        Map<String, String> mobileEmulation = new HashMap<>();
```

Comment:
Call method for creating chrome driver for mobile from BrowserManager class and remove the other unnecessary lines from this method. Thanks!

Reply:
aici nu stiu cum sa fac

Reply:
Replace all of the lines from this method with ->
driver = BrowserManager.createChromeDriverForMobile();